pmlc/dialect/tile/builder.cc

```diff
@@ -61,15 +61,15 @@ struct DomainInfo {
 };
 
 using ContractionKey = std::pair<AggregationKind, CombinationKind>;
-using Bindings = std::unordered_map<mlir::Value*, vertexai::tile::BufferPtr>;
 
 struct TileBuilder::Impl {
   MLIRContext context;
   ModuleOp module;
   OpBuilder builder;
   std::map<AffineDomainOp, DomainInfo> domains;
-  std::map<Value*, Value*> implicitUpdates;
-  Bindings implicitBindings;
+  std::unordered_map<Value*, RankedTensorType> shapeCache;
+  std::unordered_map<Value*, Value*> implicitUpdates;
+  std::unordered_map<mlir::Value*, vertexai::tile::BufferPtr> implicitBindings;
```

Comment:
Style: probably either use `Value*` or `mlir::Value*` in all of these lines, not a mix